apps/framework-cli/src/cli/routines/docker_packager.rs

```diff
@@ -72,10 +72,24 @@ impl Routine for CreateDockerfile {
 
         ensure_docker_running()?;
 
-        let file_path = internal_dir.join("packager/Dockerfile");
+        let file_path = internal_dir.join("packager/versions/.gitkeep");
```

Comment:
I don't like using shadowing when the two variables are not conceptually the same thing.

Reply:
That makes sense! Nice catch.